torch/_higher_order_ops/hop_print.py

```diff
@@ -0,0 +1,21 @@
+import torch
+import torch.utils._pytree as pytree
+from torch._ops import HigherOrderOperator
+
+
+class Print(HigherOrderOperator):
+    def __init__(self):
+        super().__init__("hop_print")
+        self._print_str = None
+
+    def __call__(self, *args, **kwargs):
+        if self._print_str is None:
+            self._print_str = self._get_print_str()
+        print(self._print_str)
+        return self._fn(*args, **kwargs)
+
```

Comment:
can you follow what is done in [other HOO](https://github.com/pytorch/pytorch/blob/52db60170dd192afa48233f03574a8d8bcd39d35/torch/_higher_order_ops/executorch_call_delegate.py#L28-L33) where we have an implementation for the [CompositeExplicitAutograd](https://github.com/pytorch/pytorch/blob/52db60170dd192afa48233f03574a8d8bcd39d35/torch/_higher_order_ops/executorch_call_delegate.py#L76-L90) dispatch key?